<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_table_by_ddl</class>
 <widget class="QWidget" name="add_table_by_ddl">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>备注：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="li_bz"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请输入建表语句</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tx_ddl"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>输出表选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="tb_output"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>效果预览:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tx_sqldll"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="bt_jiazai">
         <property name="text">
          <string>加载</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_yulan">
         <property name="text">
          <string>预览</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_submit">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_close">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
